### PR TITLE
moving onSubmit to the form

### PR DESCRIPTION
**Pull request by Haroenv**
Unless choo is doing some magic I don't yet understand, it would seem more logical to me that the *form* listens to a submit (so enter in the other input would submit too. 

Maybe that's already the case when you put the listener on the input type submit, but this looks nicer IMO

Let me know if you think differently and why :smile:

### CODE REVIEW

File: content/guides-designing-for-reusability.md [hunk 1]
```diff
@@ -104,10 +104,10 @@ const html = require('choo/html')
 
 module.exports = function formComponent (opts) {
   return `
```
Comment (yoshuawuyts) [inline content/guides-designing-for-reusability.md]: lol oops, this should have `html` prefixed though haha

**Comment by yoshuawuyts** [comment]
Hah, so we're using the form `onsubmit` event. From [w3schools](http://www.w3schools.com/jsref/event_onsubmit.asp):
```html
<form onsubmit="myFunction()">
  Enter name: <input type="text">
  <input type="submit">
</form>
```
This is actually the same thing, but then using template strings :tada:

__edit:__ I think the logic goes: you listen for the form to be submitted, the submit button is just a way to trigger the form submission.

**Comment by yoshuawuyts** [comment]
neat, thanks! :D

**Comment by Haroenv** [comment]
met plezier!

**Pull request merged**